CONTRIBUTING.md

```diff
@@ -20,10 +20,14 @@ applies to other repos as well.)
 1. [Create a fork of the numberscope/frontscope repo](./doc/working-with-git-and-github.md#create-a-fork).
 2. [Clone your fork of numberscope/frontscope](./doc/working-with-git-and-github.md#clone-a-repo).
 3. [Create a branch for your contribution](./doc/working-with-git-and-github.md#create-a-branch).
-4. [Read about basic Git operations](./doc/working-with-git-and-github.md#basic-git-operations).
+4. If you are unfamilar with them or would like a refresher,
+   [read about basic Git operations](./doc/working-with-git-and-github.md#basic-git-operations).
 5. [Push your branch to GitHub](./doc/working-with-git-and-github.md#push-a-branch).
 6. [Read Numberscope's coding principles guide](./doc/code-principles.md).
-7. [Read Numberscope's style guide](./doc/code-style.md).
+7. [Read Numberscope's style guide](./doc/code-style.md). You should also
```

Comment:
Good point. Suggestions for improvement? There is no one file to link to, at least not at the moment :-/

Reply:
I added a suggestion [here](https://github.com/numberscope/frontscope/pull/177#discussion_r1001312539).

Reply:
OK, based on your suggestion (thanks!) I have reworked this. Let me know what you think, and then as soon as we resolve the export style thing I hope this can be merged so folks can start to use the bigint utilities. Thanks!